k8s-deployment-manifest-templates/deployment-mode/daemonset/container-insights-monitoring/fluentd/fluentd.yaml

```diff
@@ -37,6 +37,8 @@ metadata:
   labels:
     k8s-app: fluentd-cloudwatch
 data:
+  kubernetes.conf: |
+    kubernetes.conf
```

Comment:
Is this change necessary? I don't think that we provide any k8s conf file.

Reply:
The contents are empty, but I need a configmap key, so created.

https://github.com/aws-samples/amazon-cloudwatch-container-insights/pull/117/files#diff-b76b71eea1fe6319075fdc91f50e39d354cd754250cbb0f3e05657f5e73ab19eR435-R440